docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+Electrolyzer
+============
+.. note::
+
+  The development of the electrolyzer model is ongoing.
+
+This is a simplified electrolyzer unit model used to approximate electrolysis performance.  With the current build, the model is simulated under the following assumptions:
+   * simulation of this unit model is only supported with the Multi-Component Aqueous Solution (MCAS) property package
```

Comment:
Would be convenient and helpful for the user to provide a hyperlink to MCAS documentation here